docker/Dockerfile

```diff
@@ -9,20 +10,23 @@ LABEL com.centurylinklabs.watchtower.lifecycle.post-update-timeout="1440"
 # Watchtower will run this script after restarting the updated container.
 LABEL com.centurylinklabs.watchtower.lifecycle.post-update="/var/www/html/docker/watchtower.post-update.sh"
 
-COPY . /var/www/html
-COPY ./docker/workers.conf /opt/docker/etc/supervisor.d/
+RUN docker-service-enable cron
+RUN docker-cronjob '* * * * * application /usr/local/bin/php /var/www/html/artisan schedule:run >> /dev/null 2>&1'
```

Comment:
why is this needed / whyd it work before?

Reply:
This is used to clear old chunk files of the package we are using https://github.com/pionl/laravel-chunk-upload

Reply:
What do you mean why did it work before? Nothing changed. I just moved it up higher to make better use of layer caching in docker.